packages/main/src/Popup.ts

```diff
@@ -272,11 +272,17 @@ abstract class Popup extends UI5Element {
 		renderFinished().then(() => {
 			this._updateMediaRange();
 		});
+
+		ResizeHandler.deregister(this, this._resizeHandler);
```

Comment:
Probably something like the code below will be better.

```
if (this.open) {
			if (!this._resizeHandlerRegistered) {
				ResizeHandler.register(this, this._resizeHandler);
				this._resizeHandlerRegistered = true;
			}
		} else if (this._resizeHandlerRegistered) {
			ResizeHandler.deregister(this, this._resizeHandler);
			this._resizeHandlerRegistered = false;
		}
```

Reply:
done